### PR TITLE
Provide a way to inject pagination to multiple response schema.

### PR DESCRIPTION
**Pull request by hslim8888**
Hi @vitalik 
I found some issues in pagination feature when using multiple response schemas.

### Motivation
- The type of multiple response schemas is different with the single schema. 
    - Input of response schema
        - single : type == Schema or List of Schema
       `@api.get("/tasks", response=List[TaskSchema])`
        - multiple : type == dict 
       `@api.post('/login', response={200: Token, 401: Message, 402: Message})`
    
    - Output of response schema
        - single : type == queryset or list
        `return Task.objects.all()`
        - multiple : type == tuple
        `return 200, Task.objects.all()`

- The difference in type of Input reponse occurs an error when using a builtin router class(`RouterPaginated`)
- The difference in type of Output reponse occurs an error when injecting pagination to result(response).
     
### Modifications & Checkpoints
- Add method check & response type check in `RouterPaginated` class.
    - I assume only GET method needs to be injected to pagination.
    - Also, status code 200 should be required if someone want to use multiple response schemas.
- Add result(response) type check in `_inject_pagination` function.
    - `paginator.paginate_queryset` can not handle items which type is tuple.
    - No need to force queryset evaluation to result['items'] since `from_orm` will take care of the rest of the process.

I must say django ninja is amazing. It makes my coding experience happy. 
Thank you very much.

### CODE REVIEW

**Comment by vitalik** [comment]
Hi @hslim8888 
Thank you for PR
Would you be able to make a test case that covers this functionality ?

File: ninja/pagination.py [hunk 1]
```diff
@@ -176,9 +176,13 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:
     def add_api_operation(
         self, path: str, methods: List[str], view_func: Callable, **kwargs: Any
     ) -> None:
```
Comment (vitalik) [inline ninja/pagination.py:178]: I'm not sure about this one - only GET method
Technically you can return paginated response on any method GET/POST/etc...

Comment (hslim8888) [reply ninja/pagination.py:178]: I just assume people usually use pagination on GET method and thought it's more efficient not to inject other methods to pagination.
But you're right. No need to limit possibilities.
I will rollback that thing.

**Comment by vitalik** [comment]
this one gets conflicts, and I'm not sure yet for the status codes...

**Pull request closed without merging**